lib/tasks/gettext.rake

```diff
@@ -62,6 +62,7 @@ namespace :gettext do
       old_pot = pot_file_path + ".old"
       File.rename(pot_file_path, old_pot)
       generate_new_pot
+      require 'open3'
```

Comment:
I'd require this at the top of the `else` block, before the renaming/copying happens, in case the require fails.